cluster/gce/gci/configure.sh

```diff
@@ -33,6 +33,13 @@ DEFAULT_CRICTL_SHA1="5c18f4e52ab524d429063b78d086dd18b894aae7"
 DEFAULT_MOUNTER_TAR_SHA="8003b798cf33c7f91320cd6ee5cec4fa22244571"
 ###
 
+# Ability to inject custom versions (Ubuntu OS images ONLY)
+# if KUBE_CONTAINERD_VERSION_OVERRIDE or KUBE_RUNC_VERSION_OVERRIDE
+# is set to empty then we do not override the version(s) and just
+# use whatever is in the default installation of containerd package
+CONTAINERD_VERSION=${KUBE_CONTAINERD_VERSION_OVERRIDE="v1.3.3"}
+RUNC_VERSION=${KUBE_RUNC_VERSION_OVERRIDE="v1.0.0-rc10"}
```

Comment:
is this the right syntax? I'm used to
```bash
CONTAINERD_VERSION=${KUBE_CONTAINERD_VERSION_OVERRIDE:-"v1.3.3"}
RUNC_VERSION=${KUBE_RUNC_VERSION_OVERRIDE:-"v1.0.0-rc10"}
```

Reply:
yep. we use the same technique in [`GOLDFLAGS`](https://github.com/kubernetes-sigs/cluster-capacity/blob/master/vendor/k8s.io/kubernetes/hack/lib/golang.sh#L780-L784) basically allows someone to clear the flags by using `GOLDFLAGS=` (environment variable is present but is set to empty scenario)